Source/SwiftLintFramework/Helpers/library_wrapper.swift

```diff
@@ -0,0 +1,195 @@
+import Foundation
+
+// MARK: - Shared Types & Functions
+
+struct DynamicLinkLibrary {
+    fileprivate let handle: UnsafeMutableRawPointer
+
+    func load<T>(symbol: String) -> T {
+        if let sym = dlsym(handle, symbol) {
+            return unsafeBitCast(sym, to: T.self)
+        }
+        let errorString = String(validatingUTF8: dlerror())
+        fatalError("Finding symbol \(symbol) failed: \(errorString ?? "unknown error")")
```

Comment:
It produces a violation:
```suggestion
        queuedFatalError("Finding symbol \(symbol) failed: \(errorString ?? "unknown error")")
```